src/database/DataTypes/Requisition.js

```diff
@@ -193,7 +193,7 @@ export class Requisition extends Realm.Object {
    * @param  {number}  months
    */
   set monthsToSupply(months) {
-    this.daysToSupply = months * 30;
+    this.daysToSupply = months * NUMBER_OF_DAYS_IN_A_MONTH;
   }
 
```

Comment:
There's also a getter at line 132 - this would also need updating, I think: 

```
 get monthsToSupply() {
    return this.daysToSupply / 30;
  }
```

Reply:
There was also default value in the schema which I updated. Wonder if I should write migration code. I think I'm going to put that in the too hard basket...........!!

Reply:
2dc540c67c700d680e1a745983ad2b98b10127d1